responders/src/UI/embc-responder/src/app/feature-components/search/evacuee-search/evacuee-search-results/evacuee-search-results.service.ts

```diff
@@ -77,8 +81,16 @@ export class EvacueeSearchResultsService {
       );
   }
 
+  public essFileExists(
+    essFile: string
```

Comment:
rename to paperFileId or externalReferenceId

Reply:
fixed!